frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/OpprettTiltaksgjennomforingContainer.tsx

```diff
@@ -486,8 +497,14 @@ export const OpprettTiltaksgjennomforingContainer = (
             placeholder="Velg oppstart"
             {...register("oppstart")}
             options={[
-              { label: "Dato", value: TiltaksgjennomforingOppstartstype.FELLES },
-              { label: "Løpende oppstart", value: TiltaksgjennomforingOppstartstype.LOPENDE },
+              {
+                label: "Dato",
```

Comment:
Hvorfor "Dato" i stedet for "Felles"?

Reply:
Fordi Fredrik tok utgangspunkt i Sanity 🤪